docs/DeveloperGuide.md

```diff
@@ -231,7 +231,38 @@ The following activity diagrams summarizes what happens when a user enters an `e
 **Aspect: Excluding b/ (Bill) in the `edit` command**
 * As the `bill` command allows us to add and subtract to the bill directly, the edit command is redundant and may cause user error if they were to replace it by accident.
 
-### MarkRoomsUnclean Feature
+### Bill Command
+
+#### Implementation:
+* The `bill` command takes in an INDEX indicating the index of the guest to edit in the current panel (starting from 1) and the `bill` field and is supported by the `BillCommandParser` that extracts the bill value.
+
+The following activity diagrams summarizes what happens when a user enters a `bill` command.
+
+![BillActivityDiagram](images/BillActivityDiagram.png)
+
+#### Design Considerations:
```

Comment:
Do you think its necessary to mention the "b/" prefix in the bill command?

Reply:
Yes, added